pinpoint/Chart.yaml

```diff
@@ -5,4 +5,10 @@ type: application
 # chart version. 
 version: 0.1.0
 # version number of the application being deployed. 
-appVersion: 2.0.4
\ No newline at end of file
+appVersion: 2.1.0
+# Dependecy for Pinpoint
+dependencies:
+    - name: mysql
+      version: 1.6.7
+      repository: https://kubernetes-charts.storage.googleapis.com
+      condition: mysql.enable
```

Comment:
If the value in `mysql.enable` in value.xml is true, will it use another image rather than `pinpointdocker/pinpoint-mysql` ?

Reply:
No. If `mysql.enable` is true, then `pinpointdocker/pinpoint-mysql` will be used.

`mysql.enable` is an option to determine whether to deploy MySQL through the helm dependency. If you want to use an external MySQL instance, you can disable MySQL installation by replacing it with `false`.

Also, If you want to use another image than pinpointdocker/pinpoint-mysql or different settings, you can set `mysql.enable` by true and change image or other settings in mysql part of `values.yaml`.